<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1218</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVTKOpenGLNativeWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>82</y>
      <width>791</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>391</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Editing</string>
    </property>
    <widget class="QLabel" name="color_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>change by basic Colors</string>
     </property>
    </widget>
    <widget class="QLabel" name="opacity_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>change Opacity</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="changeOpacity_spinBox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>380</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
    </widget>
    <widget class="QComboBox" name="changeColor_comboBox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>290</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>White</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Red</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Green</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Blue</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Black</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="colors_ComboBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>334</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="colors_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>change Color</string>
     </property>
    </widget>
    <widget class="QListWidget" name="myShapes_list">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>311</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="myShapes_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Shape to Edit or Delete :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteShape_button">
     <property name="geometry">
      <rect>
       <x>212</x>
       <y>490</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>delete Shape</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="shapes_ComboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="placeholderText">
     <string>select shape</string>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Cube</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sphere</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hemisphere</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pyramid</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cylinder</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tube</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Doughnut</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cone</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Curved Cylinder</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="shapes_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Shapes :</string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>openGLWidget</zorder>
   <zorder>shapes_ComboBox</zorder>
   <zorder>shapes_label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1218</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>qvtkopenglnativewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
